### PR TITLE
Replacing  wagtailfontawesome with wagtail-font-awesome-svg 0.0.2

### PR DESCRIPTION
**Pull request by mozaki1**

### CODE REVIEW

**Review by jacobtoppm — CHANGES_REQUESTED** [review]
I think per https://pypi.org/project/wagtail-font-awesome-svg/0.0.2/ you need to register the icons you want to use - currently they're not showing up in streamfield. The icons you need will be listed in the `Meta` classes of your custom streamfield blocks.

**Review by AshTNA — CHANGES_REQUESTED** [review]
Thanks for this @mozaki1 - I think the icons will need to be updated wherever it starts with `fa-` I think these exist still in `etna/collections/blocks.py`, `etna/home/blocks.py`, and `etna/records/blocks.py`

**Comment by mozaki1** [comment]
Hi Ash , thank you for the review, The new icons for wagtail-font-awesome-svg have slightly changed. They have changed the name of the icons also some are no longer there, 
you can check wagtail_hook.py, I have done the blocks for the insight page, you can check the admin page now after the upgrade.
before I do the other blocks I need a list of all approved icons so to hook them up in the register_icons function. you can see and browse from https://fontawesome.com/v5.15/icons?d=gallery&p=2&s=light,regular.

**Comment by AshTNA** [comment]
Thanks @mozaki1 - I will go and find some icons tomorrow morning :+1:

**Comment by AshTNA** [comment]
@mozaki1 i have added the icons :+1:

**Comment by mozaki1** [comment]
Thank you Ash , I will close this pull request and rebranch from the latest main, So to have only the changes needed rather than the long history associated with this branch.

File: etna/insights/blocks.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ class PromotedListItemBlock(blocks.StructBlock):
     url = blocks.URLBlock(required=True)
 
     class Meta:
-        icon = "fa-external-link"
+        icon = "star"
```
Comment (ababic) [inline etna/insights/blocks.py:97]: Just wanted to query this one?

**Review by ababic — APPROVED** [review]
LGTM :)

**Pull request merged**